tests/DecisionServiceTests/DecisionServiceTest.php

```diff
@@ -577,7 +577,7 @@ public function testGetVariationWithBucketingId()
         // check whitelisted variation
         $variationKey = $optlyObject->getVariation($this->experimentKey, $this->testUserIdWhitelisted, $userAttributesWithBucketingId);
         $this->assertEquals( $this->variationKeyControl, $variationKey);
-
+ 
```

Comment:
I think all of the new tests until this point, can go to OptimizelyTests or ProjectConfigTests files for better organization and maintenance.